main.go

```diff
@@ -117,10 +116,6 @@ func buildGraph(nextRefresh int64) (string, bool) {
 				time.Sleep(time.Second * 1)
 				return
 			}
-			return buildGraph(secondsToWait)
-		} else if events == nil {
-			fmt.Println("No new data available!")
-			return "", false
```

Comment:
Actually... I don't know. Probably I accidentally deleted it bashing around with Git, I'll readd this ASAP